Description: Relax phf, remove unpackaged deps
Last-Update: 2022-11-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -67,7 +67,7 @@
 version = "2.0"
 
 [dependencies.phf]
-version = "0.11"
+version = "0.10"
 
 [dependencies.pin-project-lite]
 version = "0.2"
@@ -104,10 +104,6 @@
 [dev-dependencies.env_logger]
 version = "0.9"
 
-[dev-dependencies.eui48-04]
-version = "0.4"
-package = "eui48"
-
 [dev-dependencies.eui48-1]
 version = "1.0"
 package = "eui48"
@@ -115,10 +111,6 @@
 [dev-dependencies.futures-executor]
 version = "0.3"
 
-[dev-dependencies.geo-types-06]
-version = "0.6"
-package = "geo-types"
-
 [dev-dependencies.geo-types-07]
 version = "0.7"
 package = "geo-types"
@@ -131,14 +123,6 @@
 version = "1.0"
 package = "serde_json"
 
-[dev-dependencies.smol_str-01]
-version = "0.1"
-package = "smol_str"
-
-[dev-dependencies.time-02]
-version = "0.2"
-package = "time"
-
 [dev-dependencies.time-03]
 version = "0.3"
 features = ["parsing"]
@@ -154,10 +138,6 @@
     "time",
 ]
 
-[dev-dependencies.uuid-08]
-version = "0.8"
-package = "uuid"
-
 [dev-dependencies.uuid-1]
 version = "1.0"
 package = "uuid"
@@ -171,15 +151,10 @@
 ]
 with-bit-vec-0_6 = ["postgres-types/with-bit-vec-0_6"]
 with-chrono-0_4 = ["postgres-types/with-chrono-0_4"]
-with-eui48-0_4 = ["postgres-types/with-eui48-0_4"]
 with-eui48-1 = ["postgres-types/with-eui48-1"]
-with-geo-types-0_6 = ["postgres-types/with-geo-types-0_6"]
 with-geo-types-0_7 = ["postgres-types/with-geo-types-0_7"]
 with-serde_json-1 = ["postgres-types/with-serde_json-1"]
-with-smol_str-01 = ["postgres-types/with-smol_str-01"]
-with-time-0_2 = ["postgres-types/with-time-0_2"]
 with-time-0_3 = ["postgres-types/with-time-0_3"]
-with-uuid-0_8 = ["postgres-types/with-uuid-0_8"]
 with-uuid-1 = ["postgres-types/with-uuid-1"]
 
 [badges.circle-ci]
